.changeset/tidy-days-warn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/integration': patch
```

Comment:
Sorry to be a pain but, I just realized that this package is v1 now, so it'll be a `minor` here to add a new feature, even if it's an optional one.